frontend/src/pages/secret-manager/SecretDashboardPage/components/SecretListView/SecretListView.tsx

```diff
@@ -656,7 +454,6 @@ export const SecretListView = ({
           onSaveSecret={handleSaveSecret}
           tags={wsTags}
           onCreateTag={() => handlePopUpOpen("createTag")}
-          handleSecretShare={(value: string) => handlePopUpOpen("createSharedSecret", { value })}
```

Comment:
No longer needs to be passed down. Components can simply use `useCreateSharedSecretPopup`